plugin/federated/CMakeLists.txt

```
@@ -0,0 +1,27 @@
+# gRPC needs to be installed first. See README.md.
+find_package(protobuf CONFIG REQUIRED)
```

```suggestion
find_package(Protobuf REQUIRED)
```

https://cmake.org/cmake/help/latest/module/FindProtobuf.html

---

Done.